validation-theme.js

```diff
@@ -0,0 +1,180 @@
+module.exports = async ({ github, context, core, probe }) => {
+
+    const fs = require('fs');
+    const author = context.payload.pull_request.user.login;
+    const themes = JSON.parse(fs.readFileSync('community-css-themes.json', 'utf8'));
+    const theme = themes[themes.length - 1];
+
+    const escapeHtml = (unsafe) => unsafe.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;').replace(/"/g, '&quot;').replace(/'/g, '&#039;');
+    const errors = [];
+    const addError = (error) => {
+        errors.push(`:x: ${error}`);
+        console.log('Found issue: ' + error);
+    };
+
+    const warnings = [];
+    const addWarning = (warning) => {
+        warnings.push(`:warning: ${warning}`);
+        console.log('Found issue: ' + warning);
+    }
+
+    if (context.payload.pull_request.additions <= context.payload.pull_request.deletions) {
+        // Don't run any validation checks if the user is just modifying existing theme config
+        return;
+    }
+
+    if (context.payload.pull_request.changed_files > 1) {
+        addError('You modified files other than `community-css-themes.json`.');
+    }
+
+    // Validate theme repo
+    let repoInfo = theme.repo.split('/');
+    if (repoInfo.length !== 2) {
+        addError(`It seems like you made a typo in the repository field ${theme.repo}`);
+    }
+
+    let [owner, repo] = repoInfo;
+    let manifest = {};
+    console.log(`Repo info: ${owner}/${repo}`);
+
+    if (owner.toLowerCase() !== author.toLowerCase()) {
+        addError(`The newly added entry is not at the end, or you are submitting on someone else's behalf. Last theme in the list is: ${theme.repo}`);
+    }
+    try {
+        const repository = await github.rest.repos.get({ owner, repo });
+        if(!repository.data.has_issues) {
+            addError('Your repository does not have issues enabled. Users will not be able to report bugs and request features.');
```

Comment:
Maybe:

```suggestion
            addWarning('Your repository does not have issues enabled. Users will not be able to report bugs and request features.');
```

We have never stated that this is a requirement, so I'd rather not show this as a hard error.